app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,74 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+  private List<Task> tasks = new ArrayList<>();
+  Long id =0L;
+
+
+  @GetMapping
+  @ResponseStatus(HttpStatus.OK)
+  public List<Task> taskList() {
+    return tasks;
+  }
+  
+  @GetMapping("/{id}")
+  @ResponseStatus(HttpStatus.OK)
+  public Task targetTask(@PathVariable("id") Long id) {
+    if ( findTask(id) == null){
+      ResponseEntity.status(HttpStatus.NOT_FOUND).body("");
+    }
+    return findTask(id);
+  }
+
+  @PostMapping
+  @ResponseStatus(HttpStatus.CREATED)
+  public Task create(@RequestBody Task task) throws Throwable {
+    task.setId(generateId());
+    tasks.add(task);
+    return task;
+  }
+
+
+  @RequestMapping(path="/{id}", method={RequestMethod.PATCH, RequestMethod.PUT})
+  @ResponseStatus(HttpStatus.OK)
+  public Task rewrite(@RequestBody Task rewrittenTask,@PathVariable("id") Long id) throws Throwable {
```

Comment:
이런 사소한 작업들도 생각보다 중요합니다. 포매터에 대해 조사해 보시고 자동으로 포매터가 작동하게 하거나, 커밋 전에 포매터를 작동시키는 습관을 들여 보세요. 
```suggestion
  public Task rewrite(@RequestBody Task rewrittenTask, @PathVariable("id") Long id) throws Throwable {
```

Reply:
@johngrib 넵 포맷터를 사용하였습니다. 아직 자동화는 못시켰는데, 커밋전에 자동으로 적용되도록 해보겠습니다~!